run.sh

```diff
@@ -42,6 +42,8 @@ prepare () {
   COUNTY_SUMMARIES_DIR="${API_OUTPUT_DIR}/county_summaries";
   CASE_SUMMARIES_DIR="${API_OUTPUT_DIR}/case_summary"
 
+  CAN_PREDICTIONS_DIR="${API_OUTPUT_DIR}/can_predictions/";
```

Comment:
Where did this name come from?  I don't see this first_100_counties API described in https://www.dropbox.com/scl/fi/o4bec2kz8dkcxdtabtqda/CAN-V1-API-Draft.paper?dl=0&eid=&oref=e&rlkey=f7elx3zmo6tt5s7mj5nvap9a0&t=278234600567106294963247#:t=278234600567106294963247 at all, so I'm not sure what I should expect it to look like or where I should expect it to be.

Reply:
Note that all of the existing directories (county_summaries, case_summaries) are legacy names that the website currently relies on.  Going forward we should converge on the layout / naming in the API draft doc.

Reply:
i don't really have a strong opinion on naming here, so happy to name it whatever. as for why this exists, there's #207 for it and the website wants it for the home page?

Reply:
I see.  If this is going to be legacy (and replaced with something defined in v1 API draft) then I don't care too much about the name either.  But FWIW I think everything we're producing are essentially "CAN predictions", right?  So maybe just output this file to the root `${API_OUTPUT_DIR}` or something?  🤷

Reply:
cool i updated to the county directory? i think that makes sense?